### ISSUE TITLE
Driver monitoring false positives at night

### ISSUE THREAD

**Issue by jadtec87** [commaai/openpilot#20216]
**Describe the bug**

At night the driver monitor will come on saying I am being distracted or not paying attention to the road but my eyes are looking straight forward.  It will continue to go to the orange warning even though I am still looking at the road.  No issue at day time driving.

**How to reproduce or log data**

Engage Openpilot at night, driving on the streets or highway.

**Expected behavior**

If my eyes are on the road and keeping my head in the same position, the warning shouldn't pop up.  Didn't have issue on 0.8.1 and below.

**Device/Version information (please complete the following information):**
 - Device: Comma 2
 - Dongle ID: 8ecde7e63dc1fd27
 - Route: 8ecde7e63dc1fd27\ 2021-03-01--20-16-36 
 - Timestamp: 4min 49s | about 7 times in the entire drive. 
 - Version: release2 | 0.8.2
 - Car make/model: 2018 Toyota Camry XSE V6

**Comment by ZwX1616** [comment]
Thanks for reporting! Can confirm this is a model bug. While ground truths look fine for segments in this route, both the snpe model and pytorch model have noisy outputs and think your eyes were closed most of the time at night. This could be fixed in the future with improved training/loss functions.

The older 0.8.1 model does this too, but it mistakes eyeglasses for sunglasses at night, so you happened to not get alerts from this.
